### PR TITLE
feat/add-login-and-ALC

### PR DESCRIPTION
**Pull request by tolehoai**

### CODE REVIEW

**Comment by QuocThangPhu** [comment]
I feel your code is clean

**Review by nhiv0 — COMMENTED** [review]
should remove .idea folder

File: src/Controller/ProfileController.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
```
Comment (nhiv0) [inline src/Controller/ProfileController.php:1]: i think Profile is not enough to be a Controller

File: .idea/carForRentSymfony.iml [hunk 2]
```diff
@@ -36,6 +36,42 @@
       <excludeFolder url="file://$MODULE_DIR$/vendor/symfony/string" />
       <excludeFolder url="file://$MODULE_DIR$/vendor/symfony/dependency-injection" />
       <excludeFolder url="file://$MODULE_DIR$/vendor/symfony/deprecation-contracts" />
+      <excludeFolder url="file://$MODULE_DIR$/vendor/symfony/security-bundle" />
```
Comment (bangdinhnfq) [inline .idea/carForRentSymfony.iml:39]: Pls remove this file

File: migrations/Version20220613073608.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('CREATE TABLE user (id INT AUTO_INCREMENT NOT NULL, email VARCHAR(180) NOT NULL, roles JSON NOT NULL, password VARCHAR(255) NOT NULL, UNIQUE INDEX UNIQ_8D93D649E7927C74 (email), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
+        $this->addSql('DROP TABLE vehicle');
+    }
+
+    public function down(Schema $schema): void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->addSql('CREATE TABLE vehicle (id INT AUTO_INCREMENT NOT NULL, name VARCHAR(255) CHARACTER SET utf8mb4 NOT NULL COLLATE `utf8mb4_unicode_ci`, brand VARCHAR(255) CHARACTER SET utf8mb4 NOT NULL COLLATE `utf8mb4_unicode_ci`, color VARCHAR(255) CHARACTER SET utf8mb4 NOT NULL COLLATE `utf8mb4_unicode_ci`, img VARCHAR(255) CHARACTER SET utf8mb4 NOT NULL COLLATE `utf8mb4_unicode_ci`, luggage INT NOT NULL, doors INT NOT NULL, passenger INT NOT NULL, price INT NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB COMMENT = \'\' ');
```
Comment (bangdinhnfq) [inline migrations/Version20220613073608.php:30]: When teardown should not create

File: src/Controller/ProfileController.php (hunk 1 above)
Comment (bangdinhnfq) [reply src/Controller/ProfileController.php:1]: For me I think it can be a controller

**Pull request closed without merging**